<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="7" ignored="0" total="50" passed="43">
  <reporter-output>
  </reporter-output>
  <suite name="BeaconAppLoadTest" duration-ms="3658399" started-at="2018-08-31T13:04:29Z" finished-at="2018-08-31T14:05:28Z">
    <groups>
    </groups>
    <test name="BeaconTest" duration-ms="3658399" started-at="2018-08-31T13:04:29Z" finished-at="2018-08-31T14:05:28Z">
      <class name="com.beacon.test.AppTestDeviceOne">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="setUp" is-config="true" duration-ms="48384" started-at="2018-08-31T13:03:41Z" finished-at="2018-08-31T13:04:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="65059" started-at="2018-08-31T13:04:29Z" finished-at="2018-08-31T13:05:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63041" started-at="2018-08-31T13:05:35Z" finished-at="2018-08-31T13:06:38Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="66295" started-at="2018-08-31T13:06:38Z" finished-at="2018-08-31T13:07:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="65138" started-at="2018-08-31T13:07:44Z" finished-at="2018-08-31T13:08:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="65737" started-at="2018-08-31T13:08:49Z" finished-at="2018-08-31T13:09:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="68945" started-at="2018-08-31T13:09:55Z" finished-at="2018-08-31T13:11:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="FAIL" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="126990" started-at="2018-08-31T13:11:04Z" finished-at="2018-08-31T13:13:11Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to create report]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to create report
at org.testng.Assert.fail(Assert.java:85)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:220)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Caused by: org.openqa.selenium.NoSuchElementException: An element could not be located on the page using the given search parameters.
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:24:21.231Z'
System info: host: 'basavarajs-MacBook-Pro.local', ip: '192.168.0.128', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_66'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities {app: /Users/basavaraj/Desktop/Be..., automationName: XCUITest, browserName: iOS, databaseEnabled: false, deviceName: iPhone, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: iOS, platformVersion: 11.4, takesScreenshot: true, udid: 5bebcfa8e9b85198945951f55c6..., usePrebuiltWDA: true, wdaConnectionTimeout: 240000, wdaLaunchTimeout: 240000, webStorageEnabled: false, xcodeOrgId: G56QYJ7ZRJ, xcodeSigningId: iPhone Developer}
Session ID: ea8e962a-6d2f-4c74-996a-219403c9147e
*** Element info: {Using=accessibility id, value=PhotoCapture}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:322)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:62)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at io.appium.java_client.FindsByAccessibilityId.findElementByAccessibilityId(FindsByAccessibilityId.java:37)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:187)
... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64655" started-at="2018-08-31T13:13:11Z" finished-at="2018-08-31T13:14:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="62845" started-at="2018-08-31T13:14:16Z" finished-at="2018-08-31T13:15:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63979" started-at="2018-08-31T13:15:19Z" finished-at="2018-08-31T13:16:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63547" started-at="2018-08-31T13:16:23Z" finished-at="2018-08-31T13:17:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63949" started-at="2018-08-31T13:17:26Z" finished-at="2018-08-31T13:18:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="66507" started-at="2018-08-31T13:18:30Z" finished-at="2018-08-31T13:19:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="FAIL" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="134583" started-at="2018-08-31T13:19:37Z" finished-at="2018-08-31T13:21:51Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to create report]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to create report
at org.testng.Assert.fail(Assert.java:85)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:220)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Caused by: org.openqa.selenium.NoSuchElementException: An element could not be located on the page using the given search parameters.
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:24:21.231Z'
System info: host: 'basavarajs-MacBook-Pro.local', ip: '192.168.0.128', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_66'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities {app: /Users/basavaraj/Desktop/Be..., automationName: XCUITest, browserName: iOS, databaseEnabled: false, deviceName: iPhone, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: iOS, platformVersion: 11.4, takesScreenshot: true, udid: 5bebcfa8e9b85198945951f55c6..., usePrebuiltWDA: true, wdaConnectionTimeout: 240000, wdaLaunchTimeout: 240000, webStorageEnabled: false, xcodeOrgId: G56QYJ7ZRJ, xcodeSigningId: iPhone Developer}
Session ID: ea8e962a-6d2f-4c74-996a-219403c9147e
*** Element info: {Using=accessibility id, value=PhotoCapture}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:322)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:62)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at io.appium.java_client.FindsByAccessibilityId.findElementByAccessibilityId(FindsByAccessibilityId.java:37)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:187)
... 29 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="66238" started-at="2018-08-31T13:21:52Z" finished-at="2018-08-31T13:22:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63697" started-at="2018-08-31T13:22:58Z" finished-at="2018-08-31T13:24:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63369" started-at="2018-08-31T13:24:02Z" finished-at="2018-08-31T13:25:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="65404" started-at="2018-08-31T13:25:05Z" finished-at="2018-08-31T13:26:10Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64787" started-at="2018-08-31T13:26:10Z" finished-at="2018-08-31T13:27:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64447" started-at="2018-08-31T13:27:15Z" finished-at="2018-08-31T13:28:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="FAIL" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="129550" started-at="2018-08-31T13:28:20Z" finished-at="2018-08-31T13:30:30Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to create report]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to create report
at org.testng.Assert.fail(Assert.java:85)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:220)
at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Caused by: org.openqa.selenium.NoSuchElementException: An element could not be located on the page using the given search parameters.
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:24:21.231Z'
System info: host: 'basavarajs-MacBook-Pro.local', ip: '192.168.0.128', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_66'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities {app: /Users/basavaraj/Desktop/Be..., automationName: XCUITest, browserName: iOS, databaseEnabled: false, deviceName: iPhone, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: iOS, platformVersion: 11.4, takesScreenshot: true, udid: 5bebcfa8e9b85198945951f55c6..., usePrebuiltWDA: true, wdaConnectionTimeout: 240000, wdaLaunchTimeout: 240000, webStorageEnabled: false, xcodeOrgId: G56QYJ7ZRJ, xcodeSigningId: iPhone Developer}
Session ID: ea8e962a-6d2f-4c74-996a-219403c9147e
*** Element info: {Using=accessibility id, value=Use Photo}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:322)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:62)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at io.appium.java_client.FindsByAccessibilityId.findElementByAccessibilityId(FindsByAccessibilityId.java:37)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:191)
... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64084" started-at="2018-08-31T13:30:30Z" finished-at="2018-08-31T13:31:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="62866" started-at="2018-08-31T13:31:34Z" finished-at="2018-08-31T13:32:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63457" started-at="2018-08-31T13:32:37Z" finished-at="2018-08-31T13:33:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64821" started-at="2018-08-31T13:33:40Z" finished-at="2018-08-31T13:34:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64515" started-at="2018-08-31T13:34:45Z" finished-at="2018-08-31T13:35:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64872" started-at="2018-08-31T13:35:50Z" finished-at="2018-08-31T13:36:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="FAIL" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="77189" started-at="2018-08-31T13:36:55Z" finished-at="2018-08-31T13:38:12Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to create report]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to create report
at org.testng.Assert.fail(Assert.java:85)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:220)
at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Caused by: org.openqa.selenium.NoSuchElementException: An element could not be located on the page using the given search parameters.
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:24:21.231Z'
System info: host: 'basavarajs-MacBook-Pro.local', ip: '192.168.0.128', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_66'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities {app: /Users/basavaraj/Desktop/Be..., automationName: XCUITest, browserName: iOS, databaseEnabled: false, deviceName: iPhone, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: iOS, platformVersion: 11.4, takesScreenshot: true, udid: 5bebcfa8e9b85198945951f55c6..., usePrebuiltWDA: true, wdaConnectionTimeout: 240000, wdaLaunchTimeout: 240000, webStorageEnabled: false, xcodeOrgId: G56QYJ7ZRJ, xcodeSigningId: iPhone Developer}
Session ID: ea8e962a-6d2f-4c74-996a-219403c9147e
*** Element info: {Using=accessibility id, value=Aerial coax construction composite (Coax Only)}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:322)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:62)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at io.appium.java_client.FindsByAccessibilityId.findElementByAccessibilityId(FindsByAccessibilityId.java:37)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:148)
... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63770" started-at="2018-08-31T13:38:12Z" finished-at="2018-08-31T13:39:16Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="62838" started-at="2018-08-31T13:39:16Z" finished-at="2018-08-31T13:40:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63270" started-at="2018-08-31T13:40:18Z" finished-at="2018-08-31T13:41:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64422" started-at="2018-08-31T13:41:22Z" finished-at="2018-08-31T13:42:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="66923" started-at="2018-08-31T13:42:26Z" finished-at="2018-08-31T13:43:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64141" started-at="2018-08-31T13:43:33Z" finished-at="2018-08-31T13:44:37Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="FAIL" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="139271" started-at="2018-08-31T13:44:38Z" finished-at="2018-08-31T13:46:57Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to create report]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to create report
at org.testng.Assert.fail(Assert.java:85)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:220)
at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Caused by: org.openqa.selenium.NoSuchElementException: An element could not be located on the page using the given search parameters.
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:24:21.231Z'
System info: host: 'basavarajs-MacBook-Pro.local', ip: '192.168.0.128', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_66'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities {app: /Users/basavaraj/Desktop/Be..., automationName: XCUITest, browserName: iOS, databaseEnabled: false, deviceName: iPhone, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: iOS, platformVersion: 11.4, takesScreenshot: true, udid: 5bebcfa8e9b85198945951f55c6..., usePrebuiltWDA: true, wdaConnectionTimeout: 240000, wdaLaunchTimeout: 240000, webStorageEnabled: false, xcodeOrgId: G56QYJ7ZRJ, xcodeSigningId: iPhone Developer}
Session ID: ea8e962a-6d2f-4c74-996a-219403c9147e
*** Element info: {Using=accessibility id, value=PhotoCapture}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:322)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:62)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at io.appium.java_client.FindsByAccessibilityId.findElementByAccessibilityId(FindsByAccessibilityId.java:37)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:187)
... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64728" started-at="2018-08-31T13:46:57Z" finished-at="2018-08-31T13:48:02Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63076" started-at="2018-08-31T13:48:02Z" finished-at="2018-08-31T13:49:05Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63991" started-at="2018-08-31T13:49:05Z" finished-at="2018-08-31T13:50:09Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63953" started-at="2018-08-31T13:50:09Z" finished-at="2018-08-31T13:51:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="65182" started-at="2018-08-31T13:51:13Z" finished-at="2018-08-31T13:52:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="66494" started-at="2018-08-31T13:52:18Z" finished-at="2018-08-31T13:53:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="FAIL" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="136509" started-at="2018-08-31T13:53:25Z" finished-at="2018-08-31T13:55:41Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to create report]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to create report
at org.testng.Assert.fail(Assert.java:85)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:220)
at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Caused by: org.openqa.selenium.NoSuchElementException: An element could not be located on the page using the given search parameters.
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:24:21.231Z'
System info: host: 'basavarajs-MacBook-Pro.local', ip: '192.168.0.128', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_66'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities {app: /Users/basavaraj/Desktop/Be..., automationName: XCUITest, browserName: iOS, databaseEnabled: false, deviceName: iPhone, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: iOS, platformVersion: 11.4, takesScreenshot: true, udid: 5bebcfa8e9b85198945951f55c6..., usePrebuiltWDA: true, wdaConnectionTimeout: 240000, wdaLaunchTimeout: 240000, webStorageEnabled: false, xcodeOrgId: G56QYJ7ZRJ, xcodeSigningId: iPhone Developer}
Session ID: ea8e962a-6d2f-4c74-996a-219403c9147e
*** Element info: {Using=accessibility id, value=PhotoCapture}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:322)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:62)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at io.appium.java_client.FindsByAccessibilityId.findElementByAccessibilityId(FindsByAccessibilityId.java:37)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:187)
... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64763" started-at="2018-08-31T13:55:41Z" finished-at="2018-08-31T13:56:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63087" started-at="2018-08-31T13:56:46Z" finished-at="2018-08-31T13:57:49Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63406" started-at="2018-08-31T13:57:49Z" finished-at="2018-08-31T13:58:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="63352" started-at="2018-08-31T13:58:53Z" finished-at="2018-08-31T13:59:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64142" started-at="2018-08-31T13:59:56Z" finished-at="2018-08-31T14:01:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="66870" started-at="2018-08-31T14:01:00Z" finished-at="2018-08-31T14:02:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="FAIL" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="135447" started-at="2018-08-31T14:02:08Z" finished-at="2018-08-31T14:04:23Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Unable to create report]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Unable to create report
at org.testng.Assert.fail(Assert.java:85)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:220)
at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
Caused by: org.openqa.selenium.NoSuchElementException: An element could not be located on the page using the given search parameters.
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.13.0', revision: '2f0d292', time: '2018-06-25T15:24:21.231Z'
System info: host: 'basavarajs-MacBook-Pro.local', ip: '192.168.0.128', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.6', java.version: '1.8.0_66'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities {app: /Users/basavaraj/Desktop/Be..., automationName: XCUITest, browserName: iOS, databaseEnabled: false, deviceName: iPhone, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: iOS, platformVersion: 11.4, takesScreenshot: true, udid: 5bebcfa8e9b85198945951f55c6..., usePrebuiltWDA: true, wdaConnectionTimeout: 240000, wdaLaunchTimeout: 240000, webStorageEnabled: false, xcodeOrgId: G56QYJ7ZRJ, xcodeSigningId: iPhone Developer}
Session ID: ea8e962a-6d2f-4c74-996a-219403c9147e
*** Element info: {Using=accessibility id, value=PhotoCapture}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:422)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at io.appium.java_client.remote.AppiumCommandExecutor.execute(AppiumCommandExecutor.java:231)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:548)
at io.appium.java_client.DefaultGenericMobileDriver.execute(DefaultGenericMobileDriver.java:42)
at io.appium.java_client.AppiumDriver.execute(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.execute(IOSDriver.java:1)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:322)
at io.appium.java_client.DefaultGenericMobileDriver.findElement(DefaultGenericMobileDriver.java:62)
at io.appium.java_client.AppiumDriver.findElement(AppiumDriver.java:1)
at io.appium.java_client.ios.IOSDriver.findElement(IOSDriver.java:1)
at io.appium.java_client.FindsByAccessibilityId.findElementByAccessibilityId(FindsByAccessibilityId.java:37)
at com.beacon.test.AppTestDeviceOne.add_Incident_Report(AppTestDeviceOne.java:187)
... 28 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="add_Incident_Report()[pri:5, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="add_Incident_Report" duration-ms="64535" started-at="2018-08-31T14:04:23Z" finished-at="2018-08-31T14:05:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- add_Incident_Report -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.beacon.test.AppTestDeviceOne@7619fbe7]" name="tearDown" is-config="true" duration-ms="604" started-at="2018-08-31T14:05:28Z" finished-at="2018-08-31T14:05:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.beacon.test.AppTestDeviceOne -->
    </test> <!-- BeaconTest -->
  </suite> <!-- BeaconAppLoadTest -->
</testng-results>
